ecdsa/signing/finalize.go

```diff
@@ -90,3 +90,13 @@ func (round *finalization) Update() (bool, *tss.Error) {
 func (round *finalization) NextRound() tss.Round {
 	return nil // finished!
 }
+
+func fillTo32BytesInPlace(src []byte) []byte {
+	oriLen := len(src)
+	if oriLen < 32 {
```

Comment:
really minor : I think it would have been nicer to accept the wanted length as another input parameter to the function. 
Besides that I would also rename the function to `padToLengthBytesInPlace`

Reply:
`padToLengthBytesInPlace` +1

Reply:
thx, updated!